src/InterfaceLimits.jl

```diff
@@ -97,15 +144,21 @@ function find_neighbor_interfaces(
     return neighbors
 end
 
-function find_injector_type(gen_buses, load_buses)
-    injector_type = Dict{ACBus, Type{<: StaticInjection}}()
-    for bus in union(gen_buses, load_buses)
+function find_injector_type(
+    gen_buses::Set{ACBus},
+    load_buses::Set{ACBus},
+    hvdc_buses::Set{ACBus},
+)
+    injector_type = Dict{ACBus,Type{<:StaticInjection}}()
+    for bus in union(gen_buses, load_buses, hvdc_buses)
```

Comment:
added hvdc_buses to union to capture buses with injector type InterconnectingConverter

Reply:
I don't think we want to do this. I'm avoiding supporting DCBus and multi-terminal lines for now

Reply:
I didn't think this invovled DCBuses - just the ac buses attached to two terminal dc lines that you had grouped into the hvdc_buses variable. 

Without the hvdc_buses included in the union, no injector type will get assigned to the buses that don't have any others loads or generators, so then the add_injector_constraint(injector_type::Type{InterconnectingConverter}) will never be called, right?

Reply:
right... this is that hack

Reply:
yeah. My original comment was to note that I edited line 153 because before it read "for bus in union(gen_buses, load_buses)" which meant that the hvdc_buses weren't being included in the loop

Reply:
I just wanted to flag the change I made incase that wasn't what you had intended for line 153